code_map.py

```diff
@@ -33,7 +33,6 @@
 def plugin_loaded():
     global MAPPERS, ACTIVE, using_universal_mapper
 
-    ACTIVE = True if get_code_map_view() else False
     using_universal_mapper = True
```

Comment:
Now CodeMap starts in inactive state, this means that if you restart ST and CodeMap panel is open, it will be inactive, you must synch. Synch command is enough, not needed to close/reopen. This change has a benefit, that CodeMap doesn't start mapping files when ST is restarted and loads all files, but the drawback that it must be synched manually after start. If you don't like it, I think it's safe enough to restore old behaviour, in this point(but not when switching projects).